CHANGELOG

```diff
@@ -1,3 +1,8 @@
+
+  [Ben Wyss]
+  * Added the Mapbox-Gl Javascript library in order to render vector
+    tiles into the IRV application.
+
```

Comment:
If you think it's worth it, you might list here also other improvements provided by this PR